import-export-cli/cmd/k8s/addApi.go

```diff
@@ -0,0 +1,230 @@
+package k8s
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+	wso2v1alpha2 "github.com/wso2/k8s-api-operator/api-operator/pkg/apis/wso2/v1alpha2"
+	"github.com/wso2/product-apim-tooling/import-export-cli/box"
+	k8sUtils "github.com/wso2/product-apim-tooling/import-export-cli/operator/utils"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+var flagApiName string
+var flagSwaggerFilePath string
+var flagReplicas int
+var flagNamespace string
+var flagApiVersion string
+var flagApiMode string
+
+const AddApiCmdLiteral = "api"
+const addApiCmdShortDesc = "Handle APIs in kubernetes cluster "
+const addApiLongDesc = `Add, Update and Delete APIs in kubernetes cluster. JSON and YAML formats are accepted.
+available modes are as follows
+* kubernetes`
+const addApiExamples = utils.ProjectName + " " + K8sCmdLiteral + " add/update " + AddApiCmdLiteral +
+	` -n petstore --from-file=./Swagger.json --replicas=3 --namespace=wso2`
+
+// addApiCmd represents the api command
+var addApiCmd = &cobra.Command{
+	Use:     AddApiCmdLiteral,
+	Short:   addApiCmdShortDesc,
+	Long:    addApiLongDesc,
+	Example: addApiExamples,
+	Run: func(cmd *cobra.Command, args []string) {
+		utils.Logln(utils.LogPrefixInfo + AddApiCmdLiteral + " called")
+		handleAddApi("")
+	},
+}
+
+func handleAddApi(nameSuffix string) {
+	validateAddApiCommand()
+
+	//swaggerCmNames := make([]string, len(flagSwaggerFilePaths))
+	//balInterceptorsCmNames := make([]string, 0, len(flagSwaggerFilePaths))
+	//var javaInterceptorsCmNames []string
```

Comment:
Is there a reason to keep this commented code segment? Shall we remove it?

Reply:
Fixed in 8ac841e8cd6ffe78a452a6255072e39a0a020a17